client/client.go

```diff
@@ -187,11 +190,24 @@ func loadServerTrust(conf Config) (*x509.CertPool, error) {
 	// Read certificate file.
 	servcert, err := ioutil.ReadFile(conf.ServerCert)
 	if err != nil {
+		log.Error("/etc/mender/server.crt is inaccessible.")
```

Comment:
`log.Errorf("%s is inaccessible: %s", conf.ServerCert, err.Error())`

Reply:
Fixed!